target/linux/ath79/dts/qca9563_dlink_dap-1720-a1.dts

```diff
@@ -0,0 +1,214 @@
+
+		led_status_red: led-1 {
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			panic-indicator;
+		};
+
+		led_status_green: led-2 {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led-3 { // RSSI middle
+			color = <LED_COLOR_ID_GREEN>;
+			function = "signal"; //LED_FUNCTION_SIGNAL;
+			function-enumerator = <2>;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		led-4 { // RSSI top
+			color = <LED_COLOR_ID_GREEN>;
+			function = "signal"; //LED_FUNCTION_SIGNAL;
+			function-enumerator = <3>;
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		led-5 { // RSSI bottom
+			color = <LED_COLOR_ID_GREEN>;
+			function = "signal"; //LED_FUNCTION_SIGNAL;
+			function-enumerator = <1>;
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+		};
+
+		led-6 { // RSSI bottom
+			color = <LED_COLOR_ID_RED>;
+			function = "signal"; //LED_FUNCTION_SIGNAL;
+			function-enumerator = <1>;
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&spi {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot"; // vendor calls it `bootloader`
+				reg = <0x000000 0x40000>;
+				read-only;
+			};
+
+			partition@40000 {
+				compatible = "u-boot,env";
+				label = "u-boot-env"; // vendor calls it `bdcfg`
+				reg = <0x040000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				label = "devdata";
+				reg = <0x050000 0x10000>;
+				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					macaddr_ath10k: macaddr@94 {
+						compatible = "mac-base";
+						reg = <0x94 0x11>;
+						#nvmem-cell-cells = <1>;
+					};
+
+					macaddr_ath9k: macaddr@b0 {
+						compatible = "mac-base";
+						reg = <0xb0 0x11>;
+						#nvmem-cell-cells = <1>;
+					};
+
+					macaddr_lan: macaddr@c9 {
+						compatible = "mac-base";
+						reg = <0xc9 0x11>;
+						#nvmem-cell-cells = <1>;
+					};
+				};
+			};
+
+			partition@60000 {
+				label = "devconf";
+				reg = <0x060000 0x10000>;
+				read-only;
+			};
+
+			partition@70000 {
+				compatible = "seama";
+				label = "firmware";
+				reg = <0x070000 0xf80000>;
+			};
+
+			partition@ff0000 {
+				label = "art"; // vendor calls it `radiocfg`
+				reg = <0xff0000 0x010000>;
+				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					cal_ath9k: cal@1000 {
+						reg = <0x1000 0x440>;
+					};
+
+					cal_ath10k: cal@5000 {
+						reg = <0x5000 0x844>;
+					};
+				};
+			};
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+		reset-gpios = <&gpio 11 GPIO_ACTIVE_LOW>; // necessary?
```

Comment:
If this is the reset GPIO for the PHY please keep it in. Sometimes it is not always necessary  to reset the PHY and it is already in the right state. The generic Linux PHY framework should take care.

Reply:
Will fix. Thanks for your review.